macaroonbakery/tests/test_utils.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+from datetime import datetime
+
+import pytz
+
+from macaroonbakery.utils import cookie
+
+
+class CookieTest(TestCase):
+
+    def test_cookie_expires_naive(self):
+        timestamp = datetime.utcnow()
+        c = cookie('http://example.com', 'test', 'value', expires=timestamp)
+        self.assertEqual(
+            int((timestamp - datetime(1970, 1, 1)).total_seconds()), c.expires)
```

Comment:
The expected value should be the first parameter to assertEqual, not the second (from examples at https://docs.python.org/2/library/unittest.html and other tests in this package).

Reply:
it is?